src/hd/input.c

```diff
@@ -452,5 +526,132 @@ int test_bit(const char *str, unsigned bit)
   return (u & mask) ? 1 : 0;
 }
 
+#define FALSE 0
+#define TRUE 1
+#define test2_bit(bit,str)   (test_bit(str,bit) ? TRUE : FALSE)
+/* pointer devices */
+hd_base_classes_t test_pointers(const __u16 *id_bus,
```

Comment:
It seems that this code is adapted from systemd-udev btw. In our project we now use udev implementation instead, which seems also more up-to-date.